/usr/local/src/conda/python-3.9.19/Python/errors.c: 282 - 282
--------------------------------------------------------------------------------

/usr/local/src/conda/python-3.9.19/Python/errors.c: 301 - 302
--------------------------------------------------------------------------------

/usr/local/src/conda/python-3.9.19/Include/object.h: 628 - 630
--------------------------------------------------------------------------------

/usr/local/src/conda/python-3.9.19/Objects/typeobject.c: 1412 - 1417
--------------------------------------------------------------------------------

/usr/local/src/conda/python-3.9.19/Objects/typeobject.c: 1425 - 1433
--------------------------------------------------------------------------------

